web-app/src/lib/db/services/credit.service.ts

```diff
@@ -136,3 +112,11 @@ export async function calculateCreditCost(
   }
   return totalCreditCost;
 }
+
+export async function deleteCreditTransaction(creditTransactionId: string) {
```

Comment:
I would argue we never want to delete credit transactions ever, it should more be like a ledger. If you need to alter state it is a new action.

Also I think it is unused